### PR TITLE
Added forecasting service

### PR DESCRIPTION
**Pull request by wimpykid26**
Created new file forecast.js and edited app.js accordingly.

### CODE REVIEW

File: forecast.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    };
+
+    request({
+      url,
+      json: true
+    }, function (error, response, body) {
+      if (error) {
+        reject('Unable to fetch Weather: ', error.message);
+      } else { // Print entire JSON 'nicely': console.log(JSON.stringify(body, null, 4));
+        var location = body.city.name;
```
Comment (bmorelli25) [inline forecast.js:20]: Your code was only showing the next days temperature at midnight. I updated it to show the next 5 days at 3PM. This code needs to be refactored badly. Feel free to start a new PR for the refactoring!

File: forecast.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+var request = require('request');
+
+module.exports = function (location) {
+  return new Promise (function (resolve, reject) {
+    var apiKey = process.env.API_KEY;
+    console.log(location);
+    var url = `http://api.openweathermap.org/data/2.5/forecast?q=${encodeURIComponent(location)}&mode=json&units=imperial&APPID=${apiKey}`;
```
Comment (bmorelli25) [inline forecast.js:7]: removed the ',us' after the location to allow for searches from outside the United States

File: app.js [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
 var location = require('./location.js');
+var forecast = require('./forecast.js');
 
 var argv = require('yargs')
-  .option('location', {
-    alias: 'l',
-    demand: false,
-    describe: 'Location to fetch weather',
-    type: 'string'
+  .command(function (yargs) {
```
Comment (bmorelli25) [inline app.js:6]: Good call on leaving out the optional command. It's not needed

**Review by bmorelli25 — APPROVED** [review]
Nice job! I changed around forecast.js a little. See the comments as to why. I'm going to merge and close this PR. Thanks a lot for contributing! Feel free to open another issue and contribute more if you'd like!

**Comment by wimpykid26** [comment]
@bmorelli25 thank you for merging!

**Comment by wimpykid26** [comment]
@bmorelli25 could you kindly add the hacktoberfest label to this PR?

**Pull request merged**